samples/java-spring-cinema-booking-choreography/src/main/java/com/example/cinema/model/Show.java

```diff
@@ -0,0 +1,220 @@
+package com.example.cinema.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.vavr.Tuple2;
+import io.vavr.collection.HashMap;
+import io.vavr.collection.Map;
+import io.vavr.control.Either;
+import io.vavr.control.Option;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static com.example.cinema.model.CinemaApiModel.ShowCommandError.*;
+import static com.example.cinema.model.Show.SeatStatus.AVAILABLE;
+import static io.vavr.control.Either.left;
+import static io.vavr.control.Either.right;
+
+public record Show(String id, String title, Map<Integer, Seat> seats,
+                   Map<String, Integer> pendingReservations,
+                   Map<String, FinishedReservation> finishedReservations, int availableSeats) {
+
+    public static Show create(ShowEvent.ShowCreated showCreated) {
+        List<Tuple2<Integer, Seat>> seats = showCreated.seats().stream().map(seat -> new Tuple2<>(seat.number(), seat)).toList();
+        return new Show(showCreated.showId(), showCreated.title(), HashMap.ofEntries(seats), HashMap.empty(), HashMap.empty(), showCreated.seats().size());
+    }
+
+    public Either<CinemaApiModel.ShowCommandError, ShowEvent> process(CinemaApiModel.ShowCommand command) {
+        return switch (command) {
```

Comment:
I've noticed that formatting is different than in all other samples, I think by default we have 2 spaces for indetation:
<img width="301" alt="image" src="https://github.com/lightbend/kalix-jvm-sdk/assets/2954053/3bea30fc-91bc-474b-a41e-14a22be0ee95">
@octonato do you confirm?

Reply:
Yes, we use indent 2 for the samples.